kubernetes/samples/scripts/terraform/terraformrc

```diff
@@ -0,0 +1,4 @@
+# Example of terraform plugin file
```

Comment:
missing copyright notice.. not sure this file is needed, would normally be a per-installation file.